tanguyantoine/react-native-music-control
# Add high-resolution assets for android notifications

#### What's this PR do?

Adds high-res android resources for notification tray.

#### Which issue(s) is it related to?

For issue #64.

#### How to test:

Build to high-dpi device/emulator (screenshot above is from Pixel emulator), ensure icons in tray are correctly padded (same as before) but are crisp.

#### Screenshots (if appropriate)

![screenshot_1495053538](https://cloud.githubusercontent.com/assets/189568/26175116/7357d896-3b06-11e7-9271-76929476c0b3.png)
